res/metainfo.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>dev.edfloreshz.CosmicTweaks</id>
-  <name>Tweaks for COSMIC</name>
```

Comment:
I'm not sure about this, I'd like people to know this is exclusively for COSMIC.

Reply:
That's still indicated through the summary, but I can change it back if you'd like.

Reply:
Alright, let's do it, if there's confusion or anything we can always change it back.